mathgenerator/mathgen.py

```diff
@@ -43,11 +43,11 @@ def multiplicationFunc(maxRes = 99, maxMulti = 99):
     solution = str(c)
     return problem, solution
 
-def divisionFunc(maxRes = 99, maxDivid = 99):
```

Comment:
When you created moduloFunc() you deleted divisionFunc(). This must be restored before I merge. Otherwise, looks good.